docs/site/BelongsTo-relation.md

```diff
@@ -0,0 +1,180 @@
+---
+lang: en
+title: 'belongsTo Relation'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/BelongsTo-relation.html
+---
+
+## Overview
+
+A `belongsTo` relation denotes a many-to-one connection of a model to another
+model through referential integrity. The referential integrity is enforced by a
+foreign key constraint on the source model which usually references a primary
+key on the target model. This relation indicates that each instance of the
+declaring or source model belongs to exactly one instance of the target model.
```

Comment:
hmm why it's `declaring or source` here? I understand the source model, but wondering what does "declaring model" mean..

Reply:
I have copied this from HasMany docs.

My understanding: "declaring model" and "source models" are referring to the same thing - the model class where the relation is declared.

I am happy to improve this sentence, what do you propose?